README.md

```diff
@@ -1 +1,146 @@
-# Portfolio_setup_and_mobile_first
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div aling="center">
+
+  <img src="images/MoliBravoLogo.png" alt="logo" width="140"  height="auto" aling= "center"/>
+  <br/>
+
+  <h3><b>Portfolio setup and mobile first</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
```

Comment:
- [ ] [Optional] Great job! Your Readme file looks good. But I notice that you have an FAQ header in the "table of contents" but there is no section in your Readme file with "FAQ". Please consider removing it. 🙏 By doing so, you will significantly improve the readability and professionalism of your README file.